usaspending_api/database_scripts/job_archive/backfill_primary_place_of_performance_scope.py

```diff
@@ -0,0 +1,174 @@
+"""
+Jira Ticket Number(s): DEV-4120
+
+    Backfill missing place_of_performance_scope values
+
+Expected CLI:
+
+    $ python3 usaspending_api/database_scripts/job_archive/backfill_primary_place_of_performance_scope.py
+
+Purpose:
+
+    Fetch place_of_performance_scope from Broker table:
+        - published_award_financial_assistance
+    And populate FABS transaction table columns:
+        - place_of_performance_scope
+"""
+import logging
+import math
+import psycopg2
+import time
+
+from os import environ
+
+
+# DEFINE THESE ENVIRONMENT VARIABLES BEFORE RUNNING!
+SPENDING_CONNECTION_STRING = environ["DATABASE_URL"]
+BROKER_CONNECTION_STRING = environ["DATA_BROKER_DATABASE_URL"]
+
+
+BROKER_FABS_SELECT_SQL = """
+SELECT
+    published_award_financial_assistance_id,
+    place_of_performance_scope
+FROM
+    published_award_financial_assistance
+WHERE
+    published_award_financial_assistance_id BETWEEN {min} AND {max}
+"""
+
+SPENDING_FABS_UPDATE_SQL = """
+UPDATE
+    transaction_fabs AS fabs SET
+        place_of_performance_scope = broker.place_of_performance_scope
+FROM
+    (VALUES {}) AS broker(
+        published_award_financial_assistance_id,
+        place_of_performance_scope
+    )
+WHERE
+    fabs.published_award_financial_assistance_id = broker.published_award_financial_assistance_id
+    AND (
+        fabs.place_of_performance_scope IS DISTINCT FROM broker.place_of_performance_scope
+    )
+RETURNING fabs.transaction_id
```

Comment:
I don't see us using this return value anywhere.  Think there's any efficiency to be gained if Postgres doesn't have to keep track of ids to return?